### ISSUE TITLE
Code getting stuck

### ISSUE THREAD

**Issue by ry85** [NVIDIA/waveglow#236]
Hi,

Thanks for sharing the code.

I am trying to run the code, but it if getting stuck at the data_loader step.

I tried to debug it, and found it is basically getting stuck at line 91-95 in tacotron2/stft.py.

forward_transform = F.conv1d(
            input_data,
            Variable(self.forward_basis, requires_grad=False),
            stride=self.hop_length,
            padding=0)

can you tell me what might be the reason. (I am using Pytorch=1.0.0.).

**Comment by ry85** [comment]
ok, I ran it on a different machine and it worked fine.

**Comment by ry85** [comment]
ok, I ran it on a different machine and it worked fine.
